src/ocean/sys/Epoll.d

```diff
@@ -722,3 +758,108 @@ struct Epoll
         return .close(this.fd);
     }
 }
+
+debug (EpollFdSanity)
```

Comment:
It's slightly odd that this struct is defined here but isn't used in this module. Maybe put this in a separate module?

Reply:
Maybe `ocean.io.select.selector.EpollFdSanity`?

Reply:
Yeah, I'll move it to a separate module.

Reply:
Done.